### DISCUSSION TITLE
Cylinder in a Channel Benchmark

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#23781] [Q&A Meshing] [answered]
I am modeling a cylinder in a rectangular channel fluid flow field. I am using the bias_y command on the mesh near the walls to capture the wall boundary layer velocities. How can I bias the mesh on both the upper and lower walls of the fluid flow domain?
```
[Mesh]
  [basic_mesh]
  type = GeneratedMeshGenerator
  dim = 2
  xmin = 0 # left is 3, inlet
  xmax = 2.2 # right is 1, outlet
  ymin = -0.2 # bottom is 0
  ymax = 0.21 # top is 2
  nx = 100
  ny = 15
  elem_type = QUAD4
  bias_x = 1.009
  bias_y = 1.09
  []
[]
```

**Comment by GiudGiud** [comment, marked as answer]
Hello

you can use multiple GeneratedMeshGenerators and use stitchedmeshgenerators to stitch them together

or use the cartesianMG and specify every region 

Guillaume

**Reply by TLWise** [reply]
Sounds like a very complex methods in comparison to defining bias on a face. Thanks for the information.

**Reply by GiudGiud** [reply]
Is this a 2D or 3D mesh?
There's actually good biasing options for 3D for a cylinder
